### PR TITLE
Allow setting frontend-agent-id

### PR DESCRIPTION
**Pull request by juberti**

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -70,3 +70,7 @@ build-docs:
 # Deploy docs to Github Pages.
 publish-docs:
     poetry run mkdocs gh-deploy --force
+
+# Run the fixie CLI.
+cli *FLAGS:
```
Comment (hessamb) [inline Justfile:75]: No need for this. We can do `poetry run fixie ...` in this repo and it picks the latest.

File: fixieai/cli/session/commands.py [hunk 2]
```diff
@@ -18,11 +18,12 @@ def list_sessions(ctx):
 
 
 @session.command("new", help="Creates a new session and opens it.")
+@click.option("-a", "--agent", required=False)
```
Comment (hessamb) [inline fixieai/cli/session/commands.py:21]: Please add a callback that verifies agent exists and exists gracefully if not. See [auth_force_callback as an example](https://github.com/fixie-ai/fixie-sdk/blob/6691d392c500081d8b46ea24a8ed5fe71e5d022a/fixieai/cli/auth/commands.py#L8).

**Review by hessamb — APPROVED** [review]
Nice PR! couple of nits

File: Justfile (hunk 1 above)
Comment (juberti) [reply Justfile:75]: how about I make this `just fixie` which does what you say above?

Comment (hessamb) [reply Justfile:75]: sure that'd be easier to type

File: fixieai/cli/session/commands.py (hunk 2 above)
Comment (juberti) [reply fixieai/cli/session/commands.py:21]: ok, will give it a shot

**Pull request merged**